bokehjs/src/lib/core/util/svg.ts

```diff
@@ -1012,44 +1012,51 @@ export class SVGRenderingContext2D /*implements CanvasRenderingContext2D*/ {
     return this.__ctx.measureText(text)
   }
 
-  arc(x: number, y: number, radius: number, startAngle: number, endAngle: number, counterClockwise: boolean = false): void {
-    if (!isFinite(x + y + radius + startAngle + endAngle))
-      return
-    // in canvas no circle is drawn if no angle is provided.
-    if (startAngle === endAngle) {
+  arc(x: number, y: number, radius: number, start_angle: number, end_angle: number, counterclockwise: boolean = false): void {
+    this.ellipse(x, y, radius, radius, 0, start_angle, end_angle, counterclockwise)
+  }
+
+  ellipse(x: number, y: number, radius_x: number, radius_y: number, rotation: number, start_angle: number, end_angle: number, counterclockwise: boolean = false): void {
+    if (!isFinite(x + y + radius_x + radius_y + rotation + start_angle + end_angle) || isNaN(x + y + radius_x + radius_y + rotation + start_angle + end_angle))
```

Comment:
It's sufficient to only use `isFinite()`, as it returns `false` for `NaN`. We use this idiom across bokehjs, especially in glyphs.

```node
$ node
Welcome to Node.js v14.15.4.
> isFinite(NaN)
false
```